pkgs/development/compilers/idris2/default.nix

```
@@ -39,6 +42,7 @@ stdenv.mkDerivation rec {
     homepage = https://github.com/idris-lang/Idris2;
     license = stdenv.lib.licenses.bsd3;
     maintainers = with stdenv.lib.maintainers; [ wchresta ];
+    platforms = [ "x86_64-linux" "i686-linux" "x86_64-darwin" ];
```

Could this be a more specific class, like:
```suggestion
    platforms = stdenv.lib.platforms.unix;
```
or, maybe,
```suggestion
    inherit (chez.meta) platforms;
```